### DISCUSSION TITLE
리눅스 커널에 기여해보자 | 아카이브-로그

### DISCUSSION THREAD

**Discussion by giscus** [jongwooo/blog#210] [Comment]
# 리눅스 커널에 기여해보자 | 아카이브-로그

리눅스 커널은 오픈소스 프로젝트로 개발되고 있습니다.
커널 개발은 작업한 내용을 패치 형태로 이메일을 통해 제출하는 방식으로 진행됩니다.
이 글에서는 실제로 주석 오탈자를 수정하고 패치 코드를 제출해보며 리눅스 커널에 기여하는 방법에 대해 간단히 소개합니다.

https://thearchivelog.dev/article/lets-contribute-to-linux-kernel/?fbclid=PAAaYWdrSFKmFmK4Y28iVgc3XjtpY5txja6gaBzkpCTjsfFDjsyN53OPvqD_g

**Comment by hanyeonhee** [comment]
종우 요새 블로깅 성실하게 하넹 👍👍👍

**Reply by jongwooo** [reply]
@hanyeonhee 이제 자주 써야지🤩
